pillar/edx/init.sls

```diff
@@ -30,4 +30,6 @@ schedule:
     days: 21
     function: state.sls
     args:
-      - edx.run_ansible pillar="{'edx': {'ansible_flags':  '--tags install:configuration'}}"
+      - edx.run_ansible
+      kwargs:
```

Comment:
Indentation. Also, I"m not positive that this is the correct syntax so it's worth double checking.